src/Symfony/Component/Serializer/Encoder/YamlEncoder.php

```diff
@@ -46,6 +47,10 @@ public function encode($data, $format, array $context = array())
     {
         $context = array_merge($this->defaultContext, $context);
 
+        if (isset($context['array_objects'])) {
```

Comment:
can you add a new const for this new key, just as in #28709?

Reply:
I've added this const in two different places, which I think follows the same style as #28709.

Reply:
Maybe it makes sense to add this as part of the default context you are adding as well?